### ISSUE TITLE
feat:  show the last commit message for git files

### ISSUE THREAD

**Issue by Jacques-z** [eza-community/eza#876]
Just like what github shows files with their last commit msg. I love it.

I've searched man and issues with keyword `git` and `commit` but found nothing about it.
Will you guys consider adding this feature?

**Comment by daviessm** [comment]
That sounds cool, how do you think it would look? Another field on the right of the `--long` view?

**Comment by Jacques-z** [comment]
Sorry but I have no idea QAQ
I just want see it, wherever
Maybe just before the filename?
~~Or, to be crazier, offer an option --format like `git format`~~

**Comment by bew** [comment]
Another field on the right of the long format 👍
And I think it should be opt-in, with a `--git-last-commit` flag or similar

**Comment by daviessm** [comment]
> I think it should be opt-in, with a `--git-last-commit` flag or similar

Yeah definitely! I might have a look at this when I get some time (and after clap is merged) but if anyone else wants to do it first, go for it 😎
